dashboard/public/pixelation/pixelation.js

```diff
@@ -476,3 +535,5 @@ function startOverConfirmed() {
   drawGraph();
   formatBitDisplay();
 }
+
+pixelationInit();
```

Comment:
Would it make sense to wrap everything in an anonymous function, and then explicitly export only those things that we intentionally want to export? I think I've maybe said this before, so apologies if I'm repeating myself.